code-editor/auto-stop-idle/README.md

```diff
@@ -50,6 +50,45 @@ aws sagemaker update-domain \
     }'
 
 ```
+## Steps for installing in Internet-Free VPC environments:
```

Comment:
The private VPC takes more complexity. 
I would suggest to separate this support. 
Step #1 - address the cron issue in current PR. Leave a comment in readme. In doing this, we can quickly close the current PR which already adds value for customers. 
Step #2 - follow up on PR #2 to provide private VPC support. Currently, in the readme providing manual steps provide sub-optimal user experience. We need more time to simplify the UX of private VPC case.

Reply:
Removed steps for private VPC support and added a note in the readme file. As suggested will work on this as a separate PR after simplifying the UX of private VPC case.